lms/djangoapps/experiments/utils.py

```diff
@@ -150,16 +244,26 @@ def get_experiment_user_metadata_context(course, user):
                 complete_enrollment = False
                 total_courses = None
                 courses = program.get('courses')
+                remaining_courses_price = None
+                remaining_courses_purchase_url = None
                 if courses is not None:
                     total_courses = len(courses)
                     complete_enrollment = is_enrolled_in_all_courses_in_program(courses, user_enrollments)
 
+                    if PROGRAM_PRICE_FLAG.is_enabled():
+                        unenrolled_courses = get_unenrolled_courses_in_program(courses, user_enrollments)
+                        complete_enrollment = (len(unenrolled_courses) == 0)
+                        remaining_courses_price, remaining_skus = get_program_price_and_skus(unenrolled_courses)
```

Comment:
is this code saying the remaining price is the price of the courses the user is not enrolled in?  
the user could be enrolled in all the courses but not have upgraded in any of them, i'm not sure whether that is captured in this logic

Reply:
Good point. I'll update the PR